src/tag.ts

```diff
@@ -43,6 +44,9 @@ export default class GlideCoreTag extends LitElement {
 
   static override styles = styles;
 
+  @property({ reflect: true })
+  color?: 'green' | 'indigo' | 'red';
```

Comment:
Let me know how you think of `variant`. But I think of it as of naming a change or group of changes for which there's otherwise not a good name. Here, however, we have a name that explicitly describes the state change in question.

Reply:
Yeah, I agree.

Looks like on design's side they refer to it as `style` (which I don't think is great either and won't work for us anyway). Maybe we can work with them in a future iteration to align more closely with us, or we can use a different naming convention altogether here and in Figma - I have this work on my list soon™.